sips/sip-011/sip-011_ustx-marketplace-standard.md

```diff
@@ -0,0 +1,427 @@
+
+It is recommended to marketplaces to do the following if a user lists something at 1 STX, don't send 1000000 to list-in-ustx, but rather 1000000/(1+comm) as the price argument.
+
+Some examples of how the user would be informed as displayed in December 2022 on Gamma.io and Tradeport.xyz:
+
+**Example 1 tradeport.xyz**
+
+![image](https://user-images.githubusercontent.com/33360391/208997197-61111244-b597-46fd-9c08-559550a127aa.png)
+
+**Example 2 gamma.io**
+
+![image](https://user-images.githubusercontent.com/33360391/208997558-6dcbdef7-692f-4b41-b7eb-df429c3cfead.png)
+
+### Recommendations for error codes
+
+From Friedger 2022: https://github.com/stacksgov/sips/issues/60#issuecomment-1048832083
+
+unsupported token: u500
+
+insufficient balance: u103
+
+not authorized to mint: u403
+
+Werner: Are there other recommendations for error codes? What are Gamma.io and Tradeport.xyz currently using?
+
+# Using NFTs in applications
+
+Developers who wish to use a non-fungible token contract in an application should first be provided, or keep track of, various different non-fungible token implementations. When validating a non-fungible token contract, they should fetch the interface and/or source code for that contract. If the contract implements the trait, then the application can use this standard's contract interface for making transfers and getting other details defined in this standard.
+
+All of the functions in this trait return the `response` type, which is a requirement of trait definitions in Clarity. However, some of these functions should be "fail-proof", in the sense that they should never return an error. These "fail-proof" functions are those that have been recommended as read-only. If a contract that implements this trait returns an error for these functions, it may be an indication of a non-compliant contract, and consumers of those contracts should proceed with caution.
+
+## Use of Post-Conditions
+
+The Stacks blockchain includes a feature known as "Post-Conditions" or "Constraints". By defining post-conditions, users can create transactions that include pre-defined guarantees about what might happen in that contract.
+
+For example, when applications call the `transfer` function, they should _always_ use post conditions to specify that the new owner of the NFT is the recipient principal in the `transfer` function call.
+
+# Related Work
+
+Loopbom
+https://github.com/radicleart/clarity-market
+
+Megapont Ape Club
+https://explorer.stacks.co/txid/SP3D6PV2ACBPEKYJTCMH7HEN02KP87QSP8KTEH335.megapont-ape-club-nft?chain=mainnet
+
+Source at Gamma.io and or tradeport.xyz?
+##to be added##
+
+# Backwards Compatibility
+
+Not applicable
+
+# Activation
+
+This SIP is considered activated after no objections within the community before Bitcoin tip #800,000. There are already over 20 contracts deployed to mainnet that have implemented this standard. The aim with this document is to formalize it through this SIP.
```

Comment:
I think you should provide a set number of minimum contracts that have to implement the trait for activation. Could be that the number is already met, but I think having an objective threshold is a good idea.